pkg/configs/configs-fixtures/bar

```diff
@@ -0,0 +1 @@
+baz
```

Comment:
A missing line at the end of the file.

Reply:
this is on purpose - because it as body of configuration file that I'm later checking in unittests

but fixed - I added a line + updated the unittests